dawn/src/dawn/Optimizer/PassOptions.inc

```diff
@@ -61,6 +61,9 @@ OPT(bool, SetBlockSize, false, "set-block-size", "",
 OPT(bool, DataLocalityMetric, false, "data-locality-metric", "",
     "Run data-locality-metric pass group", "", false, true)
 
+OPT(bool, StencilSplitter, false, "stencil-splitter", "",
```

Comment:
if we keep the option around,should we not at least have the pass present in the gtclangastconsumer?

Reply:
Sure. I wasn't sure if we should since it was disabled. This PR was just going to make it consistent with the others and move it into PassOptions.inc. We can definitely extend the PR and re-enable the pass.